oaiharvest/harvest.py

```diff
@@ -303,6 +304,8 @@ def main(argv=None):
         except Exception as e:
             # Log error
             logger.error(str(e))
```

Comment:
In adding unittests I've dropped support for 2.6 - it's no longer supported by the core Python team, and unittests in 2.6 has too many incompatibilities